mdpow/workflows/registry.py

```diff
@@ -0,0 +1,31 @@
+# MDPOW: registry.py
+# 2023 Cade Duckworth
+
+"""
+:mod:`mdpow.workflows.registry` --- Registry of currently supported automated workflows
+=======================================================================================
+
+Each entry in :mod:`mdpow.workflows.registry` corresponds to an :class:`~mdpow.analysis.ensemble.EnsembleAnalysis`
+for which exists a corresponding automated workflow.
+
+Currently supported workflows - :mod:`~mdpow.workflows.dihedrals`
+
+Intended for use with :mod:`mdpow.workflows.base` to specify which
+:class:`~mdpow.analysis.ensemble.EnsembleAnalysis` should run iteratively over
+the provided project data directory. To include a new automated workflow for use with
+:mod:`mdpow.workflows.base`, create a key that is the name of the corresponding
+:class:`~mdpow.analysis.ensemble.EnsembleAnalysis`, with the value defined as
+`<workflow module name>.<top-level function>`.
+
+.. autodata:: registry
+
+.. seealso:: :mod:`~mdpow.workflows.base`, :mod:`~mdpow.workflows.dihedrals`
+
+"""
+
+# import analysis
+from mdpow.workflows import dihedrals
+ 
+registry = {
```

Comment:
Add documentation for the `registry`. Either as part of the code
```python
#: The `registry` contains ... 
#: New workflows are added by ...
#:    The currently available workflows are listed in  the :ref:`Table of Workflows <workflows_table>`
registry = { ... }
```
or by adding the text above when writing 
```rest
.. :data:`registry`
   The `registry` contains ...   New workflows are added by ...
   The currently available workflows are listed in  the :ref:`Table of Workflows <workflows_table>`
```

(note the hyperlink ref to the table's name)

Reply:
@orbeckst 

I can't figure this part out. I am having trouble writing the `..  data::` or `:data:` section in a way that is recognized by `.. autodata::`.

The docs are very confusing when it comes to documenting objects within a module.